app/src/main/java/net/measurementlab/ndt7/android/MainActivity.kt

```diff
@@ -1,32 +1,119 @@
 package net.measurementlab.ndt7.android
 
+import android.app.Activity
 import android.os.Handler
 import android.os.Looper
-import android.support.v7.app.AppCompatActivity
 import android.os.Bundle
 import android.util.Log
+import android.widget.TextView
 import android.widget.Toast
+import butterknife.BindView
+import butterknife.ButterKnife
+import java.net.URL
+import com.google.gson.Gson
+import net.measurementlab.ndt7.android.model.Server
+import java.util.*
```

Comment:
I'd rather avoid `*` imports, as they make the head of who's reviewing a diff (as opposed to use an IDE) spin.